spec/services/invoices/refresh_draft_and_finalize_service_spec.rb

```diff
@@ -30,7 +30,6 @@
       )
```

Comment:
Can you please also update `FinalizeJob` so that we don't generate dead job for each tax error